net-vpn/headscale/headscale-0.15.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
+
+DESCRIPTION="An open source, self-hosted implementation of the Tailscale control server"
+HOMEPAGE="https://github.com/juanfont/headscale"
+SRC_URI="
+	https://github.com/juanfont/headscale/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~jsmolic/distfiles/${P}-deps.tar.xz
+"
+LICENSE="BSD Apache-2.0 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	acct-group/headscale
+	acct-user/headscale
+"
+
+RDEPEND="
+	${DEPEND}
+	net-firewall/iptables
+"
+
+src_install() {
+	emake build
```

Comment:
"emake build", this seems suspicious, what's the purpose?

if it try to  do the compiling, then need to move to src_compile()

Reply:
This `emake build` is not needed at all, and should certainly not be in src_install phase